dracula-theme.el

```diff
@@ -17,411 +17,420 @@
+               (org-tag :foreground ,rainbow-4 :bold t :background ,bg2)
+               (org-todo :foreground ,rainbow-5 :bold t :background ,bg2)
+               (org-upcoming-deadline :foreground ,rainbow-7)
+               (org-warning :weight bold :foreground ,rainbow-4)
+               ;; outline
+               (outline-1 :foreground ,rainbow-6)
+               (outline-2 :foreground ,rainbow-3)
+               (outline-3 :foreground ,rainbow-2)
+               (outline-4 :foreground ,rainbow-5)
+               (outline-5 :foreground ,rainbow-5)
+               (outline-6 :foreground ,rainbow-8)
+               ;; powerline
+               (powerline-evil-base-face :foreground ,bg2)
+               (powerline-evil-emacs-face :inherit powerline-evil-base-face :background ,rainbow-7)
+               (powerline-evil-insert-face :inherit powerline-evil-base-face :background ,rainbow-2)
+               (powerline-evil-motion-face :inherit powerline-evil-base-face :background ,rainbow-3)
+               (powerline-evil-normal-face :inherit powerline-evil-base-face :background ,rainbow-6)
+               (powerline-evil-operator-face :inherit powerline-evil-base-face :background ,rainbow-4)
+               (powerline-evil-replace-face :inherit powerline-evil-base-face :background "#ff5555")
+               (powerline-evil-visual-face :inherit powerline-evil-base-face :background ,rainbow-5)
+               (powerline-active1 :background ,bg6 :foreground ,fg1)
+               (powerline-active2 :background ,bg6 :foreground ,fg1)
+               (powerline-inactive2 :background ,bg3 :foreground ,fg1)
+               (powerline-inactive2 :background ,bg3 :foreground ,fg1)
+               ;; rainbow-delimiters
+               (rainbow-delimiters-depth-1-face :foreground ,rainbow-1)
+               (rainbow-delimiters-depth-2-face :foreground ,rainbow-2)
+               (rainbow-delimiters-depth-3-face :foreground ,rainbow-3)
+               (rainbow-delimiters-depth-4-face :foreground ,rainbow-4)
+               (rainbow-delimiters-depth-5-face :foreground ,rainbow-5)
+               (rainbow-delimiters-depth-6-face :foreground ,rainbow-6)
+               (rainbow-delimiters-depth-7-face :foreground ,rainbow-7)
+               (rainbow-delimiters-depth-8-face :foreground ,rainbow-8)
+               (rainbow-delimiters-unmatched-face :foreground ,warning)
+               ;; rpm-spec
+               (rpm-spec-dir-face :foreground ,rainbow-6)
+               (rpm-spec-doc-face :foreground ,rainbow-4)
+               (rpm-spec-ghost-face :foreground ,rainbow-3)
+               (rpm-spec-macro-face :foreground ,rainbow-7)
+               (rpm-spec-obsolete-tag-face :inherit font-lock-warning-face)
+               (rpm-spec-package-face :foreground ,rainbow-3)
+               (rpm-spec-section-face :foreground ,rainbow-7)
+               (rpm-spec-tag-face :foreground ,rainbow-2)
+               (rpm-spec-var-face :foreground "#a0522d")
```

Comment:
Would you mind giving this custom color a label and moving it up?

Reply:
@film42 These face specs are unchanged from the current git HEAD, just moved around. We can change the PR into a generic refactoring PR if you prefer, though. You should be able to push to my `refactor` branch if you find anything you want to address in addition to tty-specific colors.

I’ll commit the changes you’ve suggested so far.